javascript-data-model/order-history.js

```diff
@@ -0,0 +1,58 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4,2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    delivered: 'Delivered Aug 8, 2020',
+    items: [{
+      type: 'book',
+      title: 'JavaScript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      price: '$31.55'
+    }]
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    shipTo: 'JS Masher',
+    returnWindow: 'Return window closed on Aug 19, 2020',
+    delivered: 'Delivered Jul 20, 2020',
+    total: '$44.53',
+    orderNumber: '113-9984268-1280257',
+    items: [{
+      type: 'book',
+      title: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      price: '$41.33'
+    }]
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    total: '$17.44',
+    shipTo: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    delivered: 'Delivered Jul 7, 2020',
+    items: [{
+      type: 'Adapter',
+      title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      returnWindow: 'Return window closed on Aug 5, 2020',
+      price: '$15.98'
+    }]
+  },
+  {
+    orderPlaced: 'July 3, 2020',
+    total: '$138.93',
+    shipTo: 'JS Masher',
+    orderNumber: '113-2883177-2648248',
+    delivered: 'Delivered Jul 5, 2020',
+    items: [{
+      type: 'Controller',
+      title: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+      price: '$94.95',
+      typeTwo: 'Book',
```

Comment:
so instead of using a typeTwo property here, we can make a separate object to contain the data for the individual items: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```